common/s2n-codec/src/decoder/checked_range.rs

```diff
@@ -76,3 +77,25 @@ impl fmt::Debug for CheckedRange {
         write!(f, "{}..{}", self.start, self.end)
     }
 }
+/// The following harnesses were automatically generated by an
+/// experimental tool developed by the Kani team.
+#[cfg(test)]
+mod bolero_harnesses {
```

Comment:
not sure about this module name... it doesn't really describe the intent of the tests much, just more of a category.

Reply:
I mainly did this so that all of the auto-generated harnesses are in a module of the same name, so I can do `cargo test bolero_harnesses` and they will all run together. 

Any suggestions on a new naming scheme? I'm also fine with renaming them for the PR but keeping `bolero_harnesses` as the generic name that gets spit out of the tool.

Reply:
I think `tests` is fine TBH